lib/ansible/modules/system/user.py

```diff
@@ -1150,7 +1151,7 @@ def create_user(self):
             if self.expires < time.gmtime(0):
                 cmd.append('0')
             else:
-                cmd.append(time.strftime(self.DATE_FORMAT, self.expires))
+                cmd.append(str(calendar.timegm(self.expires)))
```

Comment:
Use `to_native` rather than `str()`:
```suggestion
                cmd.append(to_native(calendar.timegm(self.expires)))
```

Reply:
There is no need to call `to_native` on an `int` since it ultimately uses `str()` on the value anyways. If anything the use of `str()` could be removed since it's also converted later:

https://github.com/ansible/ansible/blob/6ac22416af987b4cd3e4cadf0f1d01352daf203c/lib/ansible/modules/system/user.py#L548